perma_web/perma/models.py

```diff
@@ -215,21 +221,21 @@ def all_folder_trees(self):
         if self.is_registrar_member():
             vesting_orgs = self.registrar.vesting_orgs.all()
         else:
-            vesting_orgs = [self.get_default_vesting_org()]
+            vesting_orgs = list(self.get_default_vesting_org())
+
+
         return [self.root_folder.get_descendants(include_self=True)] + \
             ([vesting_org.shared_folder.get_descendants(include_self=True) for vesting_org in vesting_orgs if vesting_org])
 
     def get_default_vesting_org(self):
```

Comment:
It looks like this function should now be renamed to something like `get_accessible_vesting_orgs` -- that's how it's being used in `all_folder_trees`, `LinkManager.user_access_folder`, and `FolderManager.user_access_folder` where it's called.

This also means that:

- `if self.is_registrar_member():` should return `self.registrar.vesting_orgs.all()`
- `all_folder_trees`, `LinkManager.user_access_folder` and `FolderManager.user_access_folder` should drop the `is_registrar_member()` branch -- they can use the same logic for everyone